src/incidents_filtered.Rmd

```diff
@@ -0,0 +1,87 @@
+---
+title: "incidents_filtered"
+author: "Kateryna Savchyn"
+date: "6/11/2019"
```

Comment:
Date should be in YYYY-MM-DD ([ISO_8601](https://en.wikipedia.org/wiki/ISO_8601)).